remote-config.jsonc

```diff
@@ -123,6 +123,30 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "You can contribute your own tips to these daily tips with a PR on https://github.com/ddev/remote-config"
+        },
+        {
+          "message": "DDEV's goal is to make developers happy. We hope you're happy today!"
```

Comment:
"message": "DDEV's goal is to make developers happy. We hope you're happy today!"
Seems to me like
"message": "DDEV's goal is to make developers happy." - is the punchline and then something like  "for the community by the community" might close it better.

Reply:
Ooo, I oversaw that button. thanks for the tip.